<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputConfig</class>
 <widget class="QWidget" name="InputConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QwtSlider" name="slider">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtDial" name="dial">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
  </widget>
  <widget class="QwtCompass" name="compass">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>275</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnStop">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnHalf">
      <property name="text">
       <string>Half</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnFull">
      <property name="text">
       <string>Full</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="joysticks">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="open">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>310</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="joyStatus">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>310</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>160</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Axis 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Axis 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Axis2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Axis 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Axis 4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Axis 5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Axis 6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Axis 7</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="axis0">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="axis1">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="axis2">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="axis3">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="axis4">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="axis5">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="axis6">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="axis7">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtCompass</class>
   <extends>QwtDial</extends>
   <header>qwt_compass.h</header>
  </customwidget>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
  <customwidget>
   <class>QwtSlider</class>
   <extends>QWidget</extends>
   <header>qwt_slider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
